### ISSUE TITLE
Start warmup automatically 

### ISSUE THREAD

**Issue by turbopasi** [kaloczikvn/kPM#46]
The Warmup (And the beginning of a new round) now only start if someone types " !warmup " in the chat. Shouldn't this be automatic when for example enough player have joined the server ? And if there are too many ppl still missing then it should display a message "waiting for more players for warmup" or something similiar.

**Comment by kaloczikvn** [comment]
```
local s_PlayerCount = PlayerManager:GetPlayerCount()
    if self.m_GameState == GameStates.None and s_PlayerCount >= kPMConfig.MinPlayerCount then
        self:ChangeGameState(GameStates.Warmup)
    end
```

Added this logic, should auto start now when the player count is equals or greater than the kPMConfig.MinPlayerCount! The waiting for warmup text is still missing, will add it soon!

**Comment by kaloczikvn** [comment]
Fixed in 7edf1bfd59e0f6fdfa2839fcc9c88e944f4fd8a2
